src/_group.css

```diff
@@ -0,0 +1,14 @@
+.group {
```

Comment:
We can move this to the `_form.css`

Reply:
I just realised that there is already a `.grouped` class that `display:flex` form elements with padding between them:
![image](https://user-images.githubusercontent.com/5113053/64919006-f87c9280-d7a5-11e9-849e-cf05e6402b02.png)
what shall we do ?

Reply:
oh yes that's right, we can have a variant of this like `.grouped.gapless`

Reply:
Okay let's try